spring-boot/admin/src/main/java/eu/coatrack/admin/model/repository/DummyClientRegistrationRepository.java

```diff
@@ -0,0 +1,34 @@
+package eu.coatrack.admin.model.repository;
+
+/*-
+ * #%L
+ * coatrack-admin
+ * %%
+ * Copyright (C) 2013 - 2021 Corizon | Institut für angewandte Systemtechnik Bremen GmbH (ATB)
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import org.springframework.security.oauth2.client.registration.ClientRegistration;
+import org.springframework.security.oauth2.client.registration.ClientRegistrationRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class DummyClientRegistrationRepository implements ClientRegistrationRepository {
```

Comment:
This class can be removed, as a mocked ClientRegistrationRepository is instead on the Test class

Reply:
If I remove that class, I receive a `NoSuchBeanDefinitionException` when I start the Web Application. So it is mandatory to provide this bean even it is without any obvious functionality.

Reply:
So regarding this, I have told you how can you do it, as we use the same on another project ;)

Reply:
It's not that I necessarily need this bean for testing, but that the production code won't run without it.

Reply:
Indeed I tested and to run the production code, the ClientRegistrationRepository is needed. Based on this, I will verify when these specific ClientRegistrationRepository is called and maybe create a follow-up issue to encapsulate and make cleaner the Oauth2 part

Reply:
I checked and the code needs to be changed, which would result in a lot of more commits which are not directly connected to the issue of this PR. Based on this a follow-up issue #182 and this thread can be solved